webapp/cypress/integration/createBoard.ts

```diff
@@ -112,4 +112,45 @@ describe('Create and delete board / card', () => {
         cy.get('.DeleteBoardDialog button.danger').click({force: true})
         cy.contains(boardTitle).should('not.exist')
     })
+
+    it('scrolls the kanban board when dragging card to edge', () => {
+        cy.visit('/')
+
+        // Create empty boards
+        cy.log('**Create new empty board**')
+        cy.contains('+ Add board').click({force: true})
+        cy.contains('Empty board').click({force: true})
```

Comment:
Please avoid the use of force when doing an action as much as possible. I understand that it's been the pattern in other places but we'll update it soon. In here, looks like we can completely remove it but in other cases, you may add extra "guards" like visibility check (`.should('be.visible')`) before chaining an action.
The succeeding step to create group intermittently failed on me, related to "detached DOM", I'd suggest to add extra wait as well, like:
```suggestion
        cy.contains('+ Add board').should('be.visible').click()
        cy.contains('Empty board').click().wait(1000)
```